Polyhedron/demo/Polyhedron/Viewer.cpp

```diff
@@ -726,10 +727,8 @@ void Viewer::resizeGL(int w, int h)
     {
         orthoMatrix.data()[i] = (float)ortho[i];
     }
-    int max = w;
-    if (h>w)
-        max = h;
-    QVector4D length(max,max,max, 1.0);
+
+    QVector4D length(60,60,60, 1.0);
```

Comment:
Here is another hard-coded constant: 60. Probably related to the length of the arrows, in pixels.